packages/react-charting/src/components/GaugeChart/GaugeChart.base.tsx

```diff
@@ -0,0 +1,627 @@
+    const strokeWidth = 2;
+    const halfStrokeWidth = strokeWidth / 2;
+    const needleLength = this._outerRadius - this._innerRadius + EXTRA_NEEDLE_LENGTH;
+
+    return (
+      <g transform={`rotate(${rtlSafeNeedleRotation}, 0, 0)`}>
+        <path
+          d={`
+            M 0,${-halfStrokeWidth - 3}
+            L ${-needleLength},${-halfStrokeWidth - 1}
+            A ${halfStrokeWidth + 1},${halfStrokeWidth + 1},0,0,0,${-needleLength},${halfStrokeWidth + 1}
+            L 0,${halfStrokeWidth + 3}
+            A ${halfStrokeWidth + 3},${halfStrokeWidth + 3},0,0,0,0,${-halfStrokeWidth - 3}
+          `}
+          strokeWidth={strokeWidth}
+          className={this._classNames.needle}
+          transform={`translate(${-this._innerRadius + EXTRA_NEEDLE_LENGTH / 2})`}
+          data-is-focusable={true}
+          onFocus={e => this._handleFocus(e, 'Needle')}
+          onBlur={this._handleBlur}
+          onMouseEnter={e => this._handleMouseOver(e, 'Needle')}
+          onMouseMove={e => this._handleMouseOver(e, 'Needle')}
+        />
+      </g>
+    );
+  };
+
+  private _renderLegends = () => {
+    if (this.props.hideLegend) {
+      return null;
+    }
+
+    const legends: ILegend[] = this._segments.map(segment => {
+      return {
+        title: segment.legend,
+        color: segment.color!,
+        action: () => {
+          if (this.state.selectedLegend === segment.legend) {
+            this.setState({ selectedLegend: '' });
+          } else {
+            this.setState({ selectedLegend: segment.legend });
+          }
+        },
+        hoverAction: () => {
+          this.setState({ hoveredLegend: segment.legend });
+        },
+        onMouseOutAction: () => {
+          this.setState({ hoveredLegend: '' });
+        },
+      };
+    });
+
+    return <Legends legends={legends} centerLegends {...this.props.legendProps} />;
+  };
+
+  /**
+   * This function checks if the given legend is highlighted or not.
+   * A legend can be highlighted in 2 ways:
+   * 1. selection: if the user clicks on it
+   * 2. hovering: if there is no selected legend and the user hovers over it
+   */
+  private _legendHighlighted = (legend: string) => {
+    return (
+      this.state.selectedLegend === legend || (this.state.selectedLegend === '' && this.state.hoveredLegend === legend)
+    );
+  };
+
+  /**
+   * This function checks if none of the legends is selected or hovered.
+   */
+  private _noLegendHighlighted = () => {
+    return this.state.selectedLegend === '' && this.state.hoveredLegend === '';
+  };
+
+  private _handleFocus = (focusEvent: React.FocusEvent<SVGElement>, focusedElement: string) => {
+    const hoverXValue: string =
+      'Current value is ' +
+      (this.props.chartValueFormat === GaugeValueFormat.Fraction
+        ? `${((this._sweepFraction[0] / this._sweepFraction[1]) * 100).toFixed()}%`
+        : `${this._sweepFraction[0]}/${this._sweepFraction[1]}`);
+    let total = this._minValue - 1;
```

Comment:
Lets include 100 in the last segment and not leave it at 99

Reply:
fixed